apps/opik-frontend/src/components/pages/CompareExperimentsPage/CompareExperimentsPage.tsx

```diff
@@ -223,11 +227,15 @@ const DatasetCompareExperimentsPage: React.FunctionComponent = () => {
 
   const rows = useMemo(() => data?.content ?? [], [data?.content]);
   const total = data?.total ?? 0;
-  const noDataText = "There are no selected experiments";
-  const title =
-    experimentsIds.length === 1
-      ? experiment?.name
-      : `Compare (${experimentsIds.length})`;
+  const noDataText = "There are no data for selected experiments";
```

Comment:
Minor and not a blocker: We need to double check the correct writing in English:
`There are no data...` vs `There's no data`.

From the dictionary:

> USAGE
In Latin, data is the plural of datum and, historically and in specialized scientific fields, it is also treated as a plural in English, taking a plural verb, as in ‘the data were collected and classified’. In modern non-scientific use, however, it is generally not treated as a plural. Instead, it is treated as a mass noun, similar to a word like information, which takes a singular verb. Sentences such as ‘data was collected over a number of years’ are now widely accepted in standard English